pt_os_web_portal/backend/helpers/vnc.py

```diff
@@ -0,0 +1,19 @@
+import logging
+from subprocess import CalledProcessError
+
+from pitop.common.command_runner import run_command
+
+logger = logging.getLogger(__name__)
+
+
+def vnc_wpa_gui_url():
+    url = ""
+    try:
+        # Raises when a vnc session isn't active
```

Comment:
i don't think it does raise. for me when there was no vnc session, it was logging an error but returning 200
```
Jun 29 15:36:44 pi-top pt-os-web-portal[11636]: ::ffff:192.168.178.30 - - [2022-06-29 15:36:44] "GET /vnc-wpa-gui-url HTTP/1.1" 200 159 0.039026
Jun 29 15:36:44 pi-top pt-os-web-portal[11636]: error: Command '['/usr/bin/web-vnc', 'url', '--display_id', '99']' returned non-zero exit status 1.```

Reply:
`run_command` is the one that raises, not this function... if that happens, the request returns a 200 with an empty string... I've updated the comment